ppr-ui/src/composables/userAccess/useUserAccess.ts

```diff
@@ -449,7 +451,7 @@ export const useUserAccess = () => {
       submittingParty: {
         businessName: getMhrQsSubmittingParty.value.name,
         address: getMhrQsSubmittingParty.value.mailingAddress,
-        phoneNumber: getMhrQsSubmittingParty.value.phoneNumber,
+        phoneNumber: fromDisplayPhone(getMhrQsSubmittingParty.value.phoneNumber),
```

Comment:
This would keep only the digits? I had problems when the submitting party was pre-filled, then it had dashes and submission failed...

Reply:
Yep, exactly! 
It should have been formatted all along. 

This is returned from the api but the real issue was in the user access submission (specifically Manufacturers),
It was saving the first 10 digits and wasn't correctly formatted in submissions payload. 

So we were submitting `(123) 123-4567` and it would store `(123) 123-` for the owners.
More of a safe-guard here but more importantly fixed in the payload submission down below.